elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -84,12 +84,19 @@ public function filter__ep_index_name( $index_name, $blog_id, $indexables ) {
 
 	public function filter__ep_do_intercept_request( $request, $query, $args, $failures ) {
 		$fallback_error = new \WP_Error( 'vip-elasticsearch-upstream-request-failed', 'There was an error connecting to the upstream Elasticsearch server' );
+		$timeout = 1;
```

Comment:
Let's actually increase this to a default of `2` - `1` isn't _quite_ long enough for all cases.
```suggestion
		$timeout = 2;
```